### DISCUSSION TITLE
Issue building formula on linuxbrew that depends on rust

### DISCUSSION THREAD

**Discussion by georgettica** [Homebrew/discussions#4466] [Tap maintenance and Homebrew development] [answered]
I tried to create a formula [for gitoxide](https://github.com/Homebrew/homebrew-core/pull/129719) and the CI fails on linuxbrew trying to build via rust.

I would like to know how to disable linuxbrew on my specific formula so the CI will pass and the solution for this can be found without blocking existing or newly created formulas.

**Comment by p-linnane** [comment]
If Linux is a supported OS for `gitoxide` then it needs to build successfully before we can accept it.

**Reply by georgettica** [reply]
gitoxide is supported, it seems that there was some sort of dependency issue that is addressed in https://github.com/Byron/gitoxide/issues/841

@p-linnane thanks anyway :)

**Comment by georgettica** [comment, marked as answer]
this was solved by @carlocab
